logistics_project/decorators.py

```diff
@@ -127,3 +128,22 @@ def inner(self, *args, **kwargs):
                 return f(self, *args, **kwargs)
         return inner
     return wrapper
+
+
+def managed_products_required(f):
```

Comment:
is there any reason to keep the same name as the other decorator? seems like it could lead to confusion